source/vibe/http/common.d

```diff
@@ -469,20 +488,25 @@ final class ChunkedOutputStream : OutputStream {
 	{
 		assert(!m_finalized);
 		if( m_buffer.data.length > 0 ) flush();
+		ubyte[] buffer = null;
+		auto getBytes = (size_t size) {
+			if (buffer is null || buffer.length < size)
+				buffer = new ubyte[](size);
+				return buffer[0..size];
+			};
+
 		if( nbytes == 0 ){
 			while( !data.empty ){
 				auto sz = data.leastSize;
 				assert(sz > 0);
-				writeChunkSize(sz);
-				m_out.write(data, sz);
-				m_out.write("\r\n");
-				m_out.flush();
+				auto bytes = getBytes(sz);
+				data.read(bytes);
+				write(bytes);
```

Comment:
As noted above, you can ignore the comment about appending bytes then (my bad). Should I have a look at that too and post another PR (would take some time though)?